release/release.go

```diff
@@ -55,51 +58,104 @@ func main() {
 	// don't prefix log messages with anything
 	log.SetFlags(0)
 
-	var cmd string
-	var v version.Version
-	var err error
-
-	switch len(os.Args) {
-	case 1:
-		log.Fatal("please provide a subcommand")
-	case 2:
-		cmd = os.Args[1]
-		v, err = version.GetCurrent()
-		if err != nil {
-			log.Fatalf("failed to get version: %v", err)
-		}
+	app := &cli.App{
+		Commands: []*cli.Command{
+			{
+				Name: "build-archive",
+				Action: func(cCtx *cli.Context) error {
+					buildArchive()
+					return nil
+				},
+			},
+			{
+				Name: "build-packages",
+				Action: func(cCtx *cli.Context) error {
+					buildMSI()
+					buildLinuxPackages()
+					return nil
+				},
+			},
+			{
+				Name: "get-version",
+				Action: func(cCtx *cli.Context) error {
+					v, err := version.GetCurrent()
+					if err != nil {
+						log.Fatalf("Failed to get current version: %v", err)
+					}
```

Comment:
(nit) You could just return err here and the other places you log.Fatal from an Action, but it hardly matters.